packages/modal/src/Modal.stories.tsx

```diff
@@ -19,7 +19,7 @@ export const Basic = () => {
 			<Modal
 				isOpen={isModalOpen}
 				onDismiss={closeModal}
-				title="This is the title of the modal dialogue, it can span lines but should not be too long."
+				title="Modal title"
```

Comment:
Can we revert this change? I know did it for screen reader testing, but it provides some valuable content for people wanting to use the component.